aptos-move/aptos-vm/src/transaction_validation.rs

```diff
@@ -196,7 +197,11 @@ pub(crate) fn run_multisig_prologue(
         bcs::to_bytes(&payload).map_err(|_| unreachable_error.clone())?
     } else {
         // Default to empty bytes if payload is not provided.
-        bcs::to_bytes::<Vec<u8>>(&vec![]).map_err(|_| unreachable_error)?
+        if features.is_multisig_v2_fix_enabled() {
```

Comment:
Seems like duplicated code for the `AptosVM`? Can we merge two together?

Reply:
The feature-gating code seems duplicated. However, it's a temporary feature gate, and the feature gate code will be removed soon after this fix is released. I think it will be cleaned up then.

Reply:
Sounds good then!